java_src/foam/dao/EnabledAwareDAO.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.dao;
+import foam.dao.*;
+import foam.mlang.MLang.*;
+import foam.mlang.predicate.Predicate;
+import foam.mlang.order.Comparator;
+import foam.dao.Sink;
+import foam.nanos.auth.EnabledAware;
+
+public class EnabledAwareDAO
+  extends ProxyDAO
+  implements EnabledAware
```

Comment:
Doesn't need to implement EnabledAware. Only the objects it's filtering need to.